changelogs/current_change_log.md

```diff
@@ -3,90 +3,75 @@
 **Date: YYYY-MM-DD  Current non published changes**
 
 ---
-## Tables changed
+
+### Tables changed
 
 | Table           | Attributes added | Attributes removed |
 | :-              |:-                |:-                  |
-| ***visit_observation_type***   | ***visit_observation_type_id***       | ***visit_observation_type***         |
-| ***visit_observation_type***   | ***source_observation_type***       | ***source_application***         |
-| ***visit_observation_type***   | ***creation_time*** |  |
-| ***visit_observation_type***   | ***description*** |  |
-| ***visit_observation_type***   | ***display_name*** |  |
-| ***visit_observation_type***   | ***name*** |  |
-| ***visit_observation_type***   | ***primary_data_type*** |  |
-
-### Explanation
+|                 |                  |                    |
 
-***visit_observation_type.visit_observation_type*** was renamed to ***visit_observation_type_id*** to maintain consistency in primary key names
+#### Explanation
 
-***visit_observation_type.source_application*** was renamed to ***source_observation_type*** to be more explicit that it is storing the type of observation (e.g. `flowsheet`)
-
----
 
-## Additional tables
+### Additional tables
 
-Infection is recorded in the condition_type and patient_condition tables.
+Consult orders are recorded in the consultation_request, consultation_request_answer (and question) and consultation_type tables.
```

Comment:
There's an incoming change that I think we're about to merge in (just waiting for a small change in validation) for questions refactoring so may as well do that at the same time so we don't forget?

```suggestion
Consult orders are recorded in the consultation_request, consultation_type, request_answer tables.
```